### PR TITLE
Update checkout-sdk-example

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2203](https://jira.bigcommerce.com/browse/DEVDOCS-2203)

## What changed?
Removed all text that mentions editing the theme file
Added in steps for installing custom checkouts
Make a few editorial edits

### CODE REVIEW

File: docs/stencil-docs/customizing-checkout/checkout-sdk-example.md [hunk 1]
```diff
@@ -116,421 +116,24 @@ lineNumbers: true
...
-<div id="checkout-app"></div>
-```
+1. Navigate to the [Open source checkout](https://github.com/bigcommerce/checkout-js)repository. 
+2. Clone the respository.
+3. Move into the checkout-js directory.
+5. Run `npm ci` to install dependencies.
+6. Build out your custom checkout page by making changes to the source code located in the checkout.js folder. 
+7. Run `npm run build` to build the source code.
 
 ### Congratulations!
```
Comment (capsula4) [inline docs/stencil-docs/customizing-checkout/checkout-sdk-example.md:128]: this is no longer relevant, as `stencil start` is specific for theme integration..

**Comment by capsula4** [comment]
I'm tempted to remove this page altogeher, as the checkout-sdk-quickstart guide already explains how to use Open Source Checkout.

Back in the days, we have a Checkout SDK

File: docs/stencil-docs/customizing-checkout/checkout-sdk-example.md [hunk 2]
```diff
@@ -116,421 +116,24 @@ lineNumbers: true
...
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```
-<div id="checkout-app"></div>
-```
+1. Navigate to the [Open source checkout](https://github.com/bigcommerce/checkout-js)repository.
```
Comment (capsula4) [inline docs/stencil-docs/customizing-checkout/checkout-sdk-example.md:121]: I think we might want to keep the instructions of the checkout sdk example tied to the Checkout SDK Example repository. Will investigate what's required to do so.

**Comment by capsula4** [comment]
After taking a look, I think this page can be removed. Might be worth keeping the instructions inside the repo's README.

The only thing that should be required is to simply:
1. Build checkout file
2. Upload it to webdav
3. Paste webdav URL in CP Settings. 

These instructions are better suited to be kept in the github README. All dev docs pointing to this page should point instead directly to the GH repository: https://github.com/bigcommerce/checkout-sdk-js-example

File: docs/stencil-docs/customizing-checkout/checkout-sdk-example.md [hunk 3]
```diff
@@ -116,421 +116,19 @@ lineNumbers: true
 extensions: ['.js', '.jsx']
 ```
 
-### Add React Checkout Components to Theme
+### Creating a <span class ="fn">checkout-loader.js</span> file
```
Comment (uwikaiddi) [inline docs/stencil-docs/customizing-checkout/checkout-sdk-example.md:119]:
```suggestion
### Creating a checkout-loader.js file

```

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

File: docs/stencil-docs/customizing-checkout/checkout-sdk-example.md [hunk 4]
```diff
@@ -116,421 +116,20 @@ lineNumbers: true
 extensions: ['.js', '.jsx']
 ```
 
-### Add React Checkout Components to Theme
+### Creating a checkout-loader.js file
```
Comment (capsula4) [inline docs/stencil-docs/customizing-checkout/checkout-sdk-example.md:119]: We have got this ticket https://jira.bigcommerce.com/browse/CHECKOUT-5191 to provide the checkout sdk with an checkout loader file. We shouldn't need to explain how to create the checkout loader file, it should come with the code, same way the open source checkout does it today.

Only instructions are:
1. Install dependencies (npm ci)
2. Build checkout (npm build)
3. Upload checkout-loader file
4. Enter checkout-loader URL in Control Panel

Sorry for the confusion, we have been discovering missing bits and pieces along the way, and the Checkout SDK Example is one of them.

**Comment by uwikaiddi** [comment]
@traci707, does this still need to be merged, or was there a change to whether we were going through with this change?

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**